packages/block-library/src/page-list/edit.js

```diff
@@ -235,14 +235,16 @@ const PageItems = memo( function PageItems( {
 
 function ItemSubmenuToggle( { title } ) {
 	return (
-		<button
-			className="wp-block-navigation-item__content wp-block-navigation-submenu__toggle"
-			aria-expanded="false"
-		>
-			{ title }
+		<>
+			<button
+				className="wp-block-navigation-item__content wp-block-navigation-submenu__toggle"
+				aria-expanded="false"
+			>
+				{ title }
+			</button>
 			<span className="wp-block-page-list__submenu-icon wp-block-navigation__submenu-icon">
 				<ItemSubmenuIcon />
 			</span>
-		</button>
```

Comment:
This change also needs to be done for the page list front end, [here](https://github.com/WordPress/gutenberg/blob/9e793a6c7b56eb38c16647e68805b439f5703ffa/packages/block-library/src/page-list/index.php#L191). As it stands there's still a diff in the front end between open on hover and click with page list submenus:
<img width="300" alt="Screen Shot 2022-02-02 at 3 02 26 pm" src="https://user-images.githubusercontent.com/8096000/152091867-4b2c1f60-fa16-4793-a307-2108a158222e.png">
(top one opens on hover, bottom on click).

Reply:
Thank you!

I wanted to note that I missed this because it appears I can't currently a navigation menu with only a page list to open submenus on click in the interface. As of #36826, we currently hide those options unless the menu `hasSubmenus`:
<img width="607" alt="Screenshot 2022-02-02 at 09 45 38" src="https://user-images.githubusercontent.com/1204802/152121187-1097d384-c18f-469f-827f-df29ddf3e10b.png">

It seems like we should change that to simply just show those options if the menu `hasPageList` or something along those lines, it doesn't seem worth it to check if the pagelist has submenus. CC: @vcanales 

I was able to check the box in the code editor though, so I'll take a stab at addressing this now.